frontend/src/components/Home/Home.js

```diff
@@ -3,8 +3,9 @@ import React from 'react';
 class Home extends React.Component {
 
     componentDidMount() {
-        document.body.style = 'background: #986AD9; color: white;';
+        document.body.style = 'background-image: linear-gradient(to bottom right, #177b91, #63D8F2, #177b91); color: white;';
```

Comment:
I see this background image color is the same in all the files changed. Maybe instead create a css file that we can import in each file we need it?

Reply:
Tried with CSS first, did not work. Almost certainly a way to do it, but probably best to leave it however it works right now.